kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleChannelCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+
+  <KCard
+    :to="to"
+    layout="horizontal"
+    thumbnailDisplay="small"
+    thumbnailAlign="right"
+    :title="title"
+    :headingLevel="headingLevel"
+  >
+    <template #thumbnailPlaceholder>
+      <div>
+        <ChannelThumbnail
```

Comment:
`ChannelThumbnail` component contains plenty of sizing, ratio, styles, and background logic that `KCard` and `KImg` now takes care of and we don't want to override from consumers. Let's just utilize `KCard` APIs that were made to take care of this. I think I mentioned this in one of the other PRs where I elaborated on reasons for this more. I can't find it now though, so if that's just my bad memory and clarification is needed, let me know and we can chat in more detail. 

You can just put `ContentIcon` into `#thumbnailPlaceholder` and the rest of the things will be taken care of by `KCard`. If the reason it didn't work was https://github.com/learningequality/kolibri-design-system/pull/752, after you resolve the feedback I left there,  `KCard` should support all you need here.